app/controllers/account_controller.rb

```diff
@@ -1,11 +1,11 @@
-class ProfileController < ApplicationController
+class AccountController < ApplicationController
```

Comment:
Yeah that seems reasonable, I'll pluralize this (and all the partials that go with it 🙈 )